src/containers/mappage/MapInfoNational.js

```diff
@@ -0,0 +1,131 @@
+import React, { Component } from "react";
+import "./MapInfo.css";
+import { formatDateData } from "../../assets/utils/dates";
+import { nationalTotals } from "./parsingmethods/nationalParsing";
+
+class NationMapInfo extends Component {
+  render() {
+    const {
+      requestedReport,
+      firstReportDate,
+      filteredReportsByDateRange,
+      categoryDisplay,
+    } = this.props;
+    const allTotals = nationalTotals(filteredReportsByDateRange);
+    return (
+      <div className="MapInfo_Container">
+        {categoryDisplay === 0 ? (
+          <React.Fragment>
+            <div className="MapInfo_Section">
+              <div className="color-dark-blue larger-text">National</div>
+              <div className="color-light-gray">Date Range</div>
+
+              <div className="color-dark-blue small-text">
+                {formatDateData(firstReportDate)} -{" "}
+                {formatDateData(requestedReport.reportedDate)}
+              </div>
+            </div>
+            <div className="MapInfo_Section">
+              <div className="color-light-gray">Total Reports</div>
+
+              <div className="color-dark-blue larger-text">
+                {allTotals.reports}
+              </div>
+            </div>
+
+            <div className="MapInfo_Section">
+              <div className="color-light-gray">Shortages Reported</div>
+
+              <div className="color-dark-blue larger-text">
+                {allTotals.citedShortage}
+              </div>
+            </div>
+
+            <div className="MapInfo_Section">
```

Comment:
I think we can refactor this section later to DRY it up-- maybe by making an ordered array with the data that goes in this section, that includes objects with accessor and display functions for each of these things